rfc6962-bis.xml

```diff
@@ -656,7 +655,7 @@ updated on the fly.
 (see <xref target="cert_sctlist_extension"/>). This mechanism allows the use of
 unmodified TLS servers, but, because the included SCTs cannot be changed without
 re-issuing the certificate, increases the risk that the certificate will be
-refused if any of the SCTs become invalid.
+considered non-compliant if some of the SCTs become invalid (by the client not recognizing the logs they were issued from anymore).
```

Comment:
In general I prefer "non-compliant" to "non-compliant with this document". If we really need to, we could add a note at the top somewhere saying that by "non-compliant" we mean "non-compliant with this document", though I'd think that was obvious.

That said, I otherwise prefer Rob's version.